filter_block.py

```diff
@@ -45,57 +49,57 @@ def configure(self, context):
 
     def process_signals(self, signals):
         self._logger.debug("Ready to process {} signals".format(len(signals)))
-        result = signals
+        true_result = signals
+        false_result = []
         if self.conditions:
```

Comment:
Let's talk about this line tomorrow - if I specify ANY as my operator and I give no conditions, where should the signals be returned?